packages/core/admin/server/src/controllers/permission.ts

```diff
@@ -19,26 +24,27 @@ export default {
     const checkPermissionsFn = engine.checkMany(userAbility);
 
     ctx.body = {
-      data: checkPermissionsFn(input.permissions),
+      data: checkPermissionsFn(input.permissions as Permission[]),
     };
```

Comment:
If we add `satisfies Check.Response;` there is an error because data is a boolean[]. Should we modify Check.Response ?

Reply:
you are right :pepe-run: